VTEX - Marketplace APIs.json

```diff
@@ -572,19 +572,22 @@
                                                     }
                                                 }
                                         },
-                                        "Rules": {
+                                        "Rules": {                                                 
                                             "type": "object",
-                                            "description": "Items and products that belong to sellers.",
-                                            "properties": {
-                                                "Item": {
-                                                    "type": "array",
-                                                    "description": "SKUs' ID."
-                                                },
-                                                "Product": {
-                                                    "type": "array",
-                                                    "description": "Product's name."
-                                                }
-                                            }
+                                            "description": "Items and products that belong to sellers.",    
+                                            "items": {
+                                                "type": "object",          
+                                                "properties": {
+                                                                "Item": {
+                                                                    "type": "array",
+                                                                    "description": "SKUs' ID."
```

Comment:
Adicionar `items`

Reply:
Na verdade é um "object of objects", então não vou adicionar "items".